themes/default/content/blog/oidc-blog/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Managing credentials with Open ID Connect for Pulumi Deployments"
+allow_long_title: true
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2023-01-06T11:10:50-08:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: Starting today you can use Open ID Connect (OIDC) for temporary cloud credentials when using Pulumi Deployments
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - pat-gavlin
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+    - pulumi-releases
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+After [launching Pulumi Deployments](/blog/pulumi-deployments) a few months ago and receiving a ton of community feedback (thank you!) we are launching today an integration with OpenID Connect (OIDC) to enable temporary credentials and granular access controls. We would like to thank GitHub user [eriklz](https://github.com/eriklz) for creating the [original request](https://github.com/pulumi/service-requests/issues/144) for this functionality.
```

Comment:
nit: "today we are launching"

Reply:
Might also be worth mentioning that the OIDC integration is specifically for AWS, Azure, and GCP.

Reply:
> Might also be worth mentioning that the OIDC integration is specifically for AWS, Azure, and GCP.

This is a bit of a subtle point. We have OIDC integration that automatically fetches creds for those platforms, but we also support OIDC more generally by making our OIDC token available in the `PULUMI_OIDC_TOKEN` environment variable and the `/mnt/pulumi/pulumi.oidc` file. Users can consume the tokens manually in order to integrate with other platforms.

Reply:
Let me give it a re-read and see if and where it this should be captured.